app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,62 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
+    """Tests the Event model."""
+
+    def setUp(self):
+        self.future_date = datetime.strptime('07 08 2016 05 30', '%d %m %Y %H %M')
+        self.earlier_date = datetime.strptime('06 07 2016 05 30', '%d %m %Y %H %M')
+        self.timetable = Timetable.objects.create(
+            name='timtable-item',
+            code='FBI23212',
+            api_key='419223',
+            cycle_length=14,
+            current_cycle_day=2,
+            date_created=self.earlier_date,
+            date_modified=self.future_date
+        )
+
+    def test_event_create(self):
+        Event.objects.create(
+            name='some event',
+            timetable=self.timetable,
+            start_date=self.earlier_date,
+            end_date=self.future_date,
+        )
```

Comment:
You can create the event in `setUp` and leave the test case below to test that the event was indeed created and saved in the DB. This would help reduce repetition on `line 289`